Links.ascx

```diff
@@ -4,7 +4,7 @@
 <asp:Panel ID="pnlList" runat="server" CssClass="link_module" >
     <asp:Repeater ID="lstLinks" runat="server" OnItemDataBound="lstLinks_ItemDataBound">
         <HeaderTemplate>
-            <ul id="ulHeader" class="linklist <%# Horizontal %>" >
+            <ul id="ulHeader" class="linklist <%# Horizontal %>" ></ul>
```

Comment:
```suggestion
            <ul id="ulHeader" class="linklist <%# Horizontal %>" >
```

Actually, this <ul> is opened in the <HeaderTemplate> and then closed in the <FooterTemplate> so this change is not correct...